.circleci/src/jobs/@web-jobs.yml

```diff
@@ -149,7 +149,7 @@ web-test:
     - run:
         name: test
         no_output_timeout: 30m
-        command: npm run web:test -- -- --reporter=junit --outputFile=report.xml
+        command: npm run web:test -- -- run --reporter=default --reporter=junit --outputFile=report.xml
```

Comment:
do we want double --reporter here?

Reply:
Yeah - otherwise the ci fails because it doesn't see any output for too long